tests/container/container.go

```diff
@@ -0,0 +1,443 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
```

Comment:
Why is this package included in the test subdirectory? Cant it be part of `pkg/builder`

Reply:
the idea is since it is test releated builder. So wanted to keep it in a single directory, so that we can use it for any repo with minimal changes.